src/core/cgroup.c

```diff
@@ -343,6 +403,70 @@ void cgroup_context_apply(CGroupContext *c, CGroupMask mask, const char *path, M
                                        "Failed to set cpu.cfs_quota_us on %s: %m", path);
         }
 
+        if (mask & CGROUP_MASK_IO) {
+                char buf[MAX(DECIMAL_STR_MAX(uint64_t)+1,
```

Comment:
It would be nicer to define separate `buf`s in both branches, to avoid the situation where the size of the buffer gets out of sync with future changes (and it'll be easier to read too).